src/shared/spawn-polkit-agent.h

```diff
@@ -20,3 +20,9 @@ static inline int polkit_agent_open_if_enabled(
 
         return polkit_agent_open();
 }
+
+/* A dummy struct to make _cleanup_ type-safe */
+typedef struct PolkitAgent {} PolkitAgent;
+static inline void polkit_agent_closep(PolkitAgent *agent) {
+        polkit_agent_close();
+}
```

Comment:
something similar as to my pager comments applies here, but it is less essential that this kind of agent stays around that long